lib/utils.js

```diff
@@ -33,7 +33,8 @@ function debrack (s) {
 }
 
 function uriToFilename (uri, base) {
-  var filename = path.join(base, uri)
+  var decoded = uri.split('/').map(decodeURIComponent).join('/')
```

Comment:
Reversible mapping.   The file world is self-consistent.  The URL world is self-consistent.